app/common/data/models_user.py

```diff
@@ -20,9 +23,15 @@ class User(BaseModel):
     azure_ad_subject_id: Mapped[str] = mapped_column(nullable=True, unique=True)
 
     magic_links: Mapped[list["MagicLink"]] = relationship("MagicLink", back_populates="user")
+    invitations: Mapped[list["Invitation"]] = relationship(
+        "Invitation", back_populates="user", cascade="all, delete-orphan"
+    )
     roles: Mapped[list["UserRole"]] = relationship("UserRole", back_populates="user", cascade="all, delete-orphan")
     submissions: Mapped[list["Submission"]] = relationship("Submission", back_populates="created_by")
 
+    # TODO make this nullable=False once all users have a last_login_utc set
+    last_login_utc: Mapped[datetime | None] = mapped_column(nullable=True)
```

Comment:
```suggestion
    last_logged_in_at: Mapped[datetime | None] = mapped_column(nullable=True)
```

optional nit: slight tweak to the column name here just to match the `ed` suffix (login vs logged_in, we have created_at elsewhere)

Reply:
Good point, I'll tweak to `last_logged_in_at_utc` so we've got the explicit UTC as well